data/resources/non_english.json

```diff
@@ -142,6 +142,28 @@
       ],
       "description": "Challenge platform",
       "price": "Free"
+    },
+    {
+      "name": "Flu-Project",
```

Comment:
You have to sort them alphabetically. see [Format conventions](https://inventory.raw.pm/docs/docs/en/format/) & [Adding/changing content](https://inventory.raw.pm/docs/docs/en/add/#addingchanging-content)